enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/AuthFilter.java

```diff
@@ -111,6 +111,10 @@ public boolean handleRequest(RequestContext requestContext) {
         if (!canAuthenticated) {
             FilterUtils.setUnauthenticatedErrorToContext(requestContext);
         }
+        //set WWW_AUTHENTICATE header to error response
+        requestContext.addResponseHeaders(APIConstants.WWW_AUTHENTICATE, getAuthenticatorsChallengeString() +
+                ", error=\"invalid_token\"" +
+                ", error_description=\"The access token expired\"");
```

Comment:
I guess we should either remove description or provide some sort a generic message

Reply:
In APIM we use same description

Reply:
Change the description in both